src/main/webapp/javascript/user-page.js

```diff
@@ -1,9 +1,29 @@
+// loads all the recipes when the recipe loads
+async function loadRecipes() {
+  loadUserRecipes();
+}
+
+async function loadUserRecipes() {
```

Comment:
Is this currently loading user recipes correctly? It looks like you've commented out the part for rendering.

Reply:
yeah it's not rendering yet